website/cue/reference/components/sources/opentelemetry.cue

```diff
@@ -98,6 +98,28 @@ components: sources: opentelemetry: {
 		}
 	}
 
+	configuration_examples: [
+		{
+			title: "OTLP Defaults"
+			configuration: {
+				opentelemetry: {
+					grpc: {
+						address: "0.0.0.0:4317"
+						tls: {
+							enabled: true
```

Comment:
If we are enabling TLS don't we need to provide the certificate and key to use?

Reply:
Added in 18a095957